### PR TITLE
Move $wgForegroundFeatures to new wiki name T5695

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1461,6 +1461,17 @@
 	// Foreground
 	'wgForegroundFeatures' => [
 		'default' => [],
+		'marionetworkwiki' => [
+			'showActionsForAnon' => true,
+			'NavWrapperType' => 'divonly',
+			'showHelpUnderTools' => true,
+			'showRecentChangesUnderTools' => true,
+			'enableTabs' => true,
+			'wikiName' => &$GLOBALS['wgSitename'],
```
Comment (paladox) [inline LocalSettings.php:1470]: Should be $wgSiteName not globals.

**Comment by JohnFLewis** [comment]
The wiki was renamed, surely renaming the stanza from its original to its new name is better than adding it and not removing the old one?

**Comment by Amanda-Catherine** [comment]
> The wiki was renamed, surely renaming the stanza from its original to its new name is better than adding it and not removing the old one?

except that they didn't specify the former URL in the task, so I have no idea what the original name was.

**Comment by JohnFLewis** [comment]
They linked to the previous task in it

**Pull request merged**